tests/Cache/CacheRedisStoreTest.php

```diff
@@ -67,6 +67,13 @@ public function testSetMethodProperlyCallsRedis()
         $this->assertTrue($result);
     }
 
+    public function testPutMethodReturnsFalseIfTTLIsNull()
+    {
+        $redis = $this->getRedis();
```

Comment:
Both `$redis` and `$result` are redundant

Reply:
Done